src/vec-alloc.md

```diff
@@ -157,7 +155,11 @@ such we will guard against this case explicitly.
 Ok with all the nonsense out of the way, let's actually allocate some memory:
 
 ```rust,ignore
-use std::alloc::oom;
+#![feature(allocator_api, ptr_internals)]
+
+use std::alloc::{Alloc, Global, Layout, handle_alloc_error};
+use std::mem;
+use std::ptr::{NonNull, Unique};
```

Comment:
We should just advise people use NonNull; Unique is used by the stdlib internally but otherwise probably won't ever make it to stabilization.